### PR TITLE
Company crud

### PR DESCRIPTION
**Pull request by EdiOancea**

### CODE REVIEW

**Comment by EdiOancea** [comment]
[CRUD for company](https://trello.com/c/OY8nykdi/9-crud-for-company)

**Comment by lgnstn** [comment]
@EdiOancea Please resolve conflicts.

**Comment by EdiOancea** [comment]
@lgnstn updated

File: app/services/user.js [hunk 1]
```diff
@@ -15,6 +16,10 @@ class UserService extends CrudService {
       }
     }
 
+    if (user.companyId) {
```
Comment (lgnstn) [inline app/services/user.js]: Why do you have this here? When a user is created it has nothing to do with a company.

File: app/services/user.js [hunk 2]
```diff
@@ -33,6 +38,10 @@ class UserService extends CrudService {
       error.throwValidationError('You can not change the password.');
     }
 
+    if (newData.companyId) {
```
Comment (lgnstn) [inline app/services/user.js]: Same here.

File: fixtures/address.js [hunk 3]
```diff
@@ -1,6 +1,8 @@
 const createAddress = require('./utils').createAddress;
+const companyFixture = require('./company');
 
 module.exports = {
+  ...companyFixture,
```
Comment (lgnstn) [inline fixtures/address.js:5]: Why?

File: fixtures/company.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+const createCompany = require('./utils').createCompany;
+const fieldOfActivityFixture = require('./fieldOfActivity');
+
+module.exports = {
+  ...fieldOfActivityFixture,
```
Comment (lgnstn) [inline fixtures/company.js:5]: Why?

File: fixtures/user.js [hunk 5]
```diff
@@ -1,10 +1,17 @@
 const createUser = require('./utils').createUser;
+const companyFixture = require('./company');
 
 module.exports = {
+  ...companyFixture,
```
Comment (lgnstn) [inline fixtures/user.js:5]: Why?

**Review by lgnstn — CHANGES_REQUESTED** [review]

**Review by lgnstn — APPROVED** [review]

**Pull request merged**